packages/create-vite/template-svelte-ts/src/main.ts

```diff
@@ -1,7 +1,8 @@
 import './app.css'
 import App from './App.svelte'
+import { mount } from 'svelte'
```

Comment:
Perhaps we should move this line (and in the other template) to top to be consistent with the other templates.

Reply:
Aligned all templates to be:

frameworks imports -> component import -> global css import